src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java

```diff
@@ -217,7 +217,7 @@ public ActiveDirectorySecurityRealm(String domain, String site, String bindName,
 
     public ActiveDirectorySecurityRealm(String domain, String site, String bindName,
                                         String bindPassword, String server, GroupLookupStrategy groupLookupStrategy, boolean removeIrrelevantGroups, CacheConfiguration cache) {
-        this(domain, Lists.newArrayList(new ActiveDirectoryDomain(domain, server)), site, bindName, bindPassword, server, groupLookupStrategy, removeIrrelevantGroups, domain!=null, cache, true);
+        this( domain, Arrays.asList(new ActiveDirectoryDomain(domain, server)), site, bindName, bindPassword, server, groupLookupStrategy, removeIrrelevantGroups, domain!=null, cache, true);
```

Comment:
isn't this a `singletonList` ?  `java.util.Collections.singletonList(T o)`  or does this list need to be modifiable?

Reply:
no real idea if it's modified externally (I couldn't find any write access in the plugin code). But I would keep it modifiable as `Lists.newArrayList` does for backward compat and as even the field is public...